mayastor/src/bin/mayastor-client/pool_cli.rs

```diff
@@ -25,6 +25,14 @@ pub fn subcommands<'a, 'b>() -> App<'a, 'b> {
                 .multiple(true)
                 .index(2)
                 .help("Disk device files"),
+        )
+        .arg(
+            Arg::with_name("pooltype")
+                .short("t")
+                .help("the type of the pool")
+                .required(false)
+                .possible_values(&["lvs", "lvm"])
+                .default_value("lvs"),
```

Comment:
Oh and pool_cli changes look like a separate logical change to me. If so, you know what to do. :)

Reply:
These tie in with the grpc changes